TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -665,7 +665,26 @@ class RouteDetailContentViewController: UIViewController, GMSMapViewDelegate, CL
             let update = GMSCameraUpdate.fit(bounds, withPadding: mapPadding)
             mapView.animate(with: update)
         }
+    }
+
+    func centerMap(on direction: Direction, followPath: Bool = false, topHalfCentered: Bool = false) {
```

Comment:
naming a bit confusing, perhaps `createAreaBounds`

Reply:
+1 esp. regarding the arguments (what do `followPath` or `topHalfCentered` mean)